plugins/pulp_python/plugins/distributors/steps.py

```diff
@@ -76,14 +80,22 @@ def process_main(self):
             body = ElementTree.SubElement(html, 'body')
             # For each package, we need to make a reference in index.html and also make a directory
             # with its own index.html for the package
-            for name, packages in packages.items():
+            for name, packages in projects.items():
                 element = ElementTree.SubElement(body, 'a', {'href': name})
                 element.text = name
                 ElementTree.SubElement(body, 'br')
                 PublishMetadataStep._create_package_index(name, simple_path, packages)
 
             index.write(ElementTree.tostring(html, 'utf8'))
 
+        for name, packages in projects.items():
+            project_metadata_path = os.path.join(api_path, name, 'json')
+            os.makedirs(project_metadata_path)
+            project_index_metadata_path = os.path.join(project_metadata_path, 'index.html')
+            with open(project_index_metadata_path, 'w') as meta_json:
```

Comment:
Will this actually cover every case? Reading https://wiki.python.org/moin/PyPIJSON leads me to believe I should be able to get the JSON from ``project_metadata_path``, but with this setup it looks like it's only available at ``project_metadata_path/index.html``. Both work with PyPI.